src/ToricVarieties/LineBundles/properties.jl

```diff
@@ -0,0 +1,71 @@
+@doc Markdown.doc"""
+    istrivial(l::ToricLineBundle)
+
+Returns if the toric line bundle `l` is trivial.
```

Comment:
Meta remark (applies to other docstrings, and so I don't expect you to change this here): this should be reformulated as `Returns a boolean indicating wether...` or `Returns true if ...`  -- right now it sounds as if for non-trivial bundles, the function might not return at all ;-)

Reply:
Thank you. Good point. I will adjust this in a future PR.